### PR TITLE
[GH-5951] Update Authentication > Guest Access

### PR DESCRIPTION
**Pull request by matthew-src**
Happy New Year!

#### Summary

This PR updates the Guest Access section of the Authentication Configuration Settings.

#### Ticket Link

This PR partly addresses https://github.com/mattermost/docs/issues/5951.

### CODE REVIEW

**Comment by matthew-src** [comment]
There were no conflicts with this PR and #5911, but I think the metadata description of Enable Guest Access should be updated to conform to the description in this PR (perhaps without the documentation links).

**Comment by cwarnermm** [comment]
Thanks, @matthew-src - I agree. Great suggestion and please apply that update.

File: source/configure/authentication-configuration-settings.rst [hunk 1]
```diff
@@ -2540,21 +2540,19 @@ Access the following configuration settings in the System Console by going to **
...
-
-+----------------------------------------------------------------------------------------------------+
-| This feature's ``config.json`` setting is ``"Enable": false`` with options ``true`` and ``false``. |
-+----------------------------------------------------------------------------------------------------+
++---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+---------------------------------------------------------------------+
+| - **true**: Enables the guest account feature. See `Guest Accounts <https://docs.mattermost.com/onboard/guest-accounts.html>`__.                                                | - System Config path: **Authentication > Guest Access**             |
+| - **false**: **(Default)** Disables the guest account feature. See `Disable Guest Accounts <https://docs.mattermost.com/onboard/guest-accounts.html#disable-guest-accounts>`__. | - ``config.json`` setting: ``.GuestAccountsSettings.Enable: false`` |
```
Comment (cwarnermm) [inline source/configure/authentication-configuration-settings.rst:2553]: Please see https://github.com/mattermost/docs/pull/6172#discussion_r1085382223 for guidance on the format for page link references in the docs.

File: source/configure/authentication-configuration-settings.rst [hunk 2]
```diff
@@ -2588,12 +2586,11 @@ Enforce multi-factor authentication
...
+| - **true**: Guests must use MFA. Guests without MFA are redirected to the setup page until MFA is configured. | - System Config path: **Authentication > Guest Access**                                       |
+| - **false**: **(Default)** MFA is not required for guests.                                                    | - ``config.json`` setting: ``.GuestAccountsSettings.EnforceMultifactorAuthentication: false`` |
+|                                                                                                               | - Environment variable: ``MM_GUESTACCOUNTSSETTINGS_ENFORCEMULTIFACTORAUTHENTICATION``         |
++---------------------------------------------------------------------------------------------------------------+-----------------------------------------------------------------------------------------------+
+| **Note**: This setting defaults to false and cannot be changed if MFA is not enforced for non-guest users.                                                                                                    |
```
Comment (cwarnermm) [inline source/configure/authentication-configuration-settings.rst:2594]:
```suggestion
| **Note**: This setting defaults to **false** and cannot be changed if MFA isn't enforced for non-guest users.                                                                                                    |
```

**Review by cwarnermm — COMMENTED** [review]
Thanks, @matthew-src! I've provided feedback inline.

**Review by cwarnermm — APPROVED** [review]
Thanks, @matthew-src!

**Pull request merged**